lib/jekyll/converters/scss.rb

```diff
@@ -158,14 +158,14 @@ def sass_configs
 
       def convert(content)
         output = ::Sass.compile_string(content, **sass_configs)
-        result = +"#{output.css}\n"
+        result = output.css
 
         if sourcemap_required?
           source_map = process_source_map(output.source_map)
           generate_source_map_page(source_map)
 
           if (sm_url = source_mapping_url)
-            result << "\n/*# sourceMappingURL=#{sm_url} */"
+            result += "\n/*# sourceMappingURL=#{sm_url} */"
```

Comment:
I would like to have blank line before the link to the sourcemap to serve as a visual separation from CSS rules, especially when the rules are compact.

```suggestion
            result += "\n\n/*# sourceMappingURL=#{sm_url} */"
```

Reply:
@ashmaroli Updated to be consistent with dart-sass: https://github.com/sass/dart-sass/blob/1e211d640a86160910a53de600e3a55f70409470/lib/src/executable/compile_stylesheet.dart#L179-L180